test/simctl-e2e-specs.js

```diff
@@ -33,7 +33,7 @@ describe('simctl', function () {
       throw new Error('No valid SDKs');
     }
     console.log(`Found valid SDKs: ${validSdks.join(', ')}`); // eslint-disable-line no-console
-    sdk = _.last(validSdks);
+    sdk = process.env.SDK || _.last(validSdks);
```

Comment:
I would say the name of the env variable should be more specific

Reply:
Sure. I'll change it to `IOS_SDK`